core/src/com/geekbrains/td/MonsterEmitter.java

```diff
@@ -1,12 +1,47 @@
 package com.geekbrains.td;
 
 import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.math.MathUtils;
+
+import java.util.LinkedList;
 
 public class MonsterEmitter extends ObjectPool<Monster> {
     private GameScreen gameScreen;
+    private float monsterTimer;
+    private LinkedList<MonsterWaveType> waveList;
```

Comment:
Bad naming, which breaks encapsulation. Do not use implementation details for fields naming. This field holds a collection of monster waves, so call it `waves` or `monstersWaves`, or `activeWaves`.